ci/docker-rust/build.sh

```diff
@@ -8,5 +8,5 @@ docker build -t solanalabs/rust .
 read -r rustc version _ < <(docker run solanalabs/rust rustc --version)
 [[ $rustc = rustc ]]
 docker tag solanalabs/rust:latest solanalabs/rust:"$version"
-docker push solanalabs/rust:"$version"
-docker push solanalabs/rust:latest
+# docker push solanalabs/rust:"$version"
```

Comment:
Can you please just remove these from the PR.  I can then pull the PR locally and rebuild the docker image for you

Reply:
Reverted this change.